goal_src/jak1/pc/pckernel-common.gc

```diff
@@ -697,7 +697,7 @@
     (("ps2-hints?") (set! (-> obj ps2-hints?) (file-stream-read-symbol file)))
     (("ps2-lod-dist?") (set! (-> obj ps2-lod-dist?) (file-stream-read-symbol file)))
     (("force-envmap?") (set! (-> obj force-envmap?) (file-stream-read-symbol file)))
-    (("force-actors?") (set! (-> obj force-actors?) (file-stream-read-symbol file)))
+    (("ps2-actor-vis?") (set! (-> obj ps2-actor-vis?) (file-stream-read-symbol file)))
```

Comment:
Do not remove this string from the case list unless you want everyone's settings to be reset after this is merged!

Reply:
I changed both the file IO lines to use `force-actors?` in order to maintain compatibility. Is there anything else I should be aware of?

Reply:
Yes. People's settings files will have a `force-actors?`. If you removed the case, then the settings parser will assume it's an invalid key and nuke everything.

Reply:
With the changes in 1c0e5ab this should be accounted for and the parser should expect to find `force-actors?` like it has in the past no?

https://github.com/open-goal/jak-project/blob/1c0e5abb6f2374b27b1ba26a9489f552aa52c53b/goal_src/jak1/pc/pckernel-common.gc#L700

https://github.com/open-goal/jak-project/blob/1c0e5abb6f2374b27b1ba26a9489f552aa52c53b/goal_src/jak1/pc/pckernel-common.gc#L787

Reply:
I think you should:
- remove the `force-actors?` key from the file write function so it stops showing up in settings files.
- in the file read function, keep the old `force-actors?` setting but make it set the opposite value - that way people's settings aren't suddenly inverted when updating.
- add a new `ps2-actor-vis?` case that just works normally.